test/example/deployment.yaml

```diff
@@ -1,6 +1,18 @@
 # SPDX-FileCopyrightText: The RamenDR authors
 # SPDX-License-Identifier: Apache-2.0
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginxservice
+  labels:
+    app: nginx
+spec:
+  ports:
+  - port: 80
+  selector:
+    app: nginx
```

Comment:
This should match the example application - it has label app=example, so:

    app: example